packages/paste-website/src/components/shortcodes/feedback-popover/index.tsx

```diff
@@ -0,0 +1,131 @@
+import * as React from 'react';
+import {Heading} from '@twilio-paste/heading';
+import {Text} from '@twilio-paste/text';
+import {PopoverContainer, PopoverButton, Popover} from '@twilio-paste/popover';
+import {useUID} from '@twilio-paste/uid-library';
+import {Box} from '@twilio-paste/box';
+import {Separator} from '@twilio-paste/separator';
+import {Anchor} from '@twilio-paste/anchor';
+import {Button} from '@twilio-paste/button';
+import {ThumbsUpIcon} from '@twilio-paste/icons/esm/ThumbsUpIcon';
+import {ThumbsDownIcon} from '@twilio-paste/icons/esm/ThumbsDownIcon';
+import {EditIcon} from '@twilio-paste/icons/esm/EditIcon';
+import {trackCustomEvent} from 'gatsby-plugin-google-analytics';
+import {SimpleStorage} from '../../../utils/SimpleStorage';
+
+type RatingProps = {
+  likePage: (event: React.MouseEvent) => void;
+  dislikePage: (event: React.MouseEvent) => void;
+};
+
+const UnratedPage: React.FC<RatingProps> = ({likePage, dislikePage}) => (
+  <>
+    <Heading as="h4" variant="heading40">
+      Rate this page
+    </Heading>
+    <Text as="p" marginBottom="space50">
```

Comment:
Is it weird that we're not using our Paragraph here?

Reply:
The Paragraph component either has space70 or space0. For the popover, space70 looked a little too airy, as it competed with the outer padding of the whole popover. That's why I opted for this smaller margin.